server/service/src/invoice_line/outbound_shipment_unallocated_line/allocate/generate.rs

```diff
@@ -89,22 +89,22 @@ pub fn generate(
             )),
         }
 
-        remaining_to_allocate -= packs_to_allocate * stock_line.stock_line_row.pack_size;
+        remaining_to_allocate -= stock_line.stock_line_row.pack_size * packs_to_allocate as f64;
```

Comment:
This changed to f64... correct?

Reply:
Can change packs_to_allocate_from_stock_line to return f64 ?

Reply:
thanks done in:
https://github.com/msupply-foundation/open-msupply/pull/3944/commits/a864d370b674a1e5f2d7261b2e88e4292436b1bc

(sorry done it in the wrong PR...)